ledger/participant-integration-api/src/main/scala/platform/apiserver/services/admin/ApiPackageManagementService.scala

```diff
@@ -65,6 +67,7 @@ private[apiserver] final class ApiPackageManagementService private (
   override def listKnownPackages(
       request: ListKnownPackagesRequest
   ): Future[ListKnownPackagesResponse] = {
+    logger.info("Listing known packages")
```

Comment:
Should you add something to the logging context here?

Reply:
There's no identifier available in the request. Should we generate one in such case?